docs/docs-content/release-notes/release-notes.md

```diff
@@ -11,376 +11,177 @@ tags: ["release-notes"]
 
 ### Palette
 
-#### Features
+#### Breaking Changes
 
-<!-- prettier-ignore -->
-- <TpBadge /> Cluster Profile variables, a new feature that allows you to define variables in a cluster profile. This
-  feature is in Tech Preview and is available only for Edge clusters using Local UI. Profile variables allow you to define variable
-  types, apply validation, and narrow the scope of variables to a cluster profile. 
-  Check out [Cluster Profile Variables](../profiles/cluster-profiles/create-cluster-profiles/define-profile-variables.md) to learn more about
-  profile variables.
-
-- MAAS clusters using Palette eXtended Kubernetes (PXK) now support the ability to specify a custom MAAS API endpoint
-  URL and port during cluster creation. This feature allows you to use a custom DNS server or Virtual IP (VIP) that is
-  not resolvable outside of the MAAS network. Refer to the [PXK](../integrations/kubernetes.md#custom-maas-endpoint)
-  documentation for more details.
-
-- Support for [Konvoy](../integrations/konvoy.md) is now available in Palette. You can create a custom image using the
-  Konvoy image builder project and use it to deploy a Konvoy cluster. Check out the
-  [Red Hat Linux Enterprise and Konvoy](../byoos/usecases/vmware/konvoy.md) guide to learn how to create a custom image
-  and deploy a Konvoy cluster.
-
-- Multiple system administrators can now be added to the self-hosted Palette system console to help manage and maintain
-  the Palette instance. The feature helps organizations embrace the separation of duties by delegating different
-  responsibilities to system administrators. Refer to the
-  [System Administrators](../enterprise-version/system-management/account-management/account-management.md#system-administrators)
-  page to learn more about system administrators.
+- Google Cloud Platform (GCP) does not support the ability for users to specify a patch version for the Kubernetes
+  version used in GKE clusters. In this release, Palette aligns with GCP's behavior and removes the ability to specify a
+  patch version when creating a cluster profile for GKE. Only the major and minor versions are available for selection.
 
-#### Improvements
+#### Features
 
-- <TpBadge /> Nutanix cluster deployments now display YAML variables and expose them as input fields in the User
-  Interface (UI) during the cluster deployment process. Previously, the UI did not display the YAML variables for
-  Nutanix clusters and users had to update the machine template YAML manually. You can learn more about Nutanix in the
-  [Create and Manage Nutanix Cluster](../clusters/data-center/nutanix/create-manage-nutanix-cluster.md) guide.
+- The upgrade experience for MicroK8s has been improved by the introduction of new upgrade strategies. Users can now
+  choose between a RollingUpgrade, InPlaceUpgrade, or SmartUpgrade. To learn more about the new upgrade strategies,
+  refer to the [MicroK8s pack documentation](../integrations/microk8s.md).
```

Comment:
In short: we introduce upgrade support in 4.4.0, BUT we only support one upgrade strategy.
More generally, we want to mention that MicroK8s is under Tech Preview, but I don't know how to say that in the release notes.

Reply:
I added the Tech Preview badge @cloudmaniac